sonar-core/src/main/java/org/sonar/core/issue/workflow/IssueWorkflow.java

```diff
@@ -151,26 +150,19 @@ private void buildAutomaticTransitions(StateMachine.Builder builder) {
         .build())
       .transition(Transition.builder("automaticclose")
         .from(Issue.STATUS_RESOLVED).to(Issue.STATUS_CLOSED)
-        .conditions(IsBeingClosed.INSTANCE)
-        .functions(SetClosed.INSTANCE, new SetCloseDate(true))
-        .automatic()
-        .build())
-      .transition(Transition.builder("automaticclosemanual")
-        .from(Issue.STATUS_RESOLVED).to(Issue.STATUS_CLOSED)
-        .conditions(IsManual.INSTANCE)
+        .conditions(new OrCondition(IsBeingClosed.INSTANCE, IsManual.INSTANCE))
```

Comment:
was the bug that having two transitions with the same "from" and "to", only the last one was kept?
if so, some check might be wise to avoid future similar bug, WDYT?